sloop-to-eks.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+USAGE="Usage: ./sloop_to_eks.sh <cluster_name> [<region>]
+
+<cluster_name>: Provide EKS cluster to connect to.
+      <region>: defaults to us-west-2.
+"
+
+if [ $# -lt 1 ] || [ "$1" == "help" ]; then
+    echo "$USAGE"
+    exit 0
+fi
+REGION="us-west-2"
+if [ "$2" != "" ]; then
+    REGION=$2
+fi
+aws eks --region $REGION update-kubeconfig --name $1
```

Comment:
This is needed only one time, right?

I am a little hesitant to mess with the users kubeconfig.  This somewhat gets into the responsibility of setting up a workstation to talk to a specific vendor, which I am not sure belongs here.

Without update-kubeconfig this script largely becomes one command, in which case a section in the doc might be better.  Curious what others think though.

Reply:
I definitely juggle with a bunch of clusters myself and don't want to run 2 commands each time so I came up with this script.
This script talks about using Sloop with EKS (AWS) so yes it is vendor (AWS) specific.
Anyway, Sloop seems like a really useful tool.  This script attempts to make it easier to get started for AWS users.

I'm not too attached to the changes I made.  They're useful for me and I thought others might find it useful as well.  If not, I can close this PR.

Reply:
I can see the value of this, so lets keep it but put it somewhere other than root.  We already have an "other" folder that would probably work for now.  Or you can suggest something that could be use for all providers and would not be confusing.

Just out of curiosity: Why do you use the `update-kubeconfig` command each time?  I am able to have several EKS clusters in my kubeConfig and I just pass in the `--context` argument to sloop when running against one.  And alternatively I could use `kubeconfig config use-context`.  Is `update-kubeconfig` doing something different?

Reply:
@hsiddulugari thanks for contributing!
I agree with @thomashargrove - can we move helper script to `other` folder or create a new one (e.g.: `scripts` or `providers`)? It looks like client-go path forward is to remove all provider specific code from auth(https://github.com/kubernetes/client-go/tree/master/plugin/pkg/client/auth) and just use `exec` auth, so we might need to add gcp/azure scripts in the future.

Reply:
@thomashargrove ,
with `update-kubeconfig`, I'm going for simplicity again (all I have to provide is cluster name) but with `use-context`, I have to provide the full context: `arn:aws:eks:<region>:<account-number>:cluster/<cluster-name>`.  Note the extra account number in it even if we were to build the context ourselves using arguments.  I don't see any harm in doing update-kubeconfig each time.  Do you?

@thomashargrove and @nurland , how about `providers/aws`. I can move the script to that directory.